backend/account/authentication_controller.py

```diff
@@ -194,10 +194,15 @@ def set_user_organization(self, request: Request, organization_id: str) -> Respo
                 except MethodNotImplemented:
                     Logger.info("frictionless_onboarding not implemented")
 
-            if new_organization:
+                try:
+                    self.auth_service.hubspot_signup_api(request=request)
```

Comment:
Yes, @johnyrahul. It will be missing, but it will go to the `except` block where we're gracefully handling the error. This won't throw any error during the process.

Reply:
@johnyrahul Thanks for the catch. I tested it when all exceptions were being caught. However, after I changed it to specifically catch the `MethodNotFound` exception, I didn't test it for OSS alone.

Since this PR has already been merged, here's the PR where I fixed this issue: https://github.com/Zipstack/unstract/pull/383